### ISSUE TITLE
Add Option to enable other Library types

### ISSUE THREAD

**Issue by neuleo** [jarnedemeulemeester/findroid#776]
**Is your feature request related to a problem? Please describe.**
Yes, I also use Jellyfin for Media serving like Videos I edited myself or downloaded YouTube Videos. 
For Movies and Series, I only use Findroid as it's the absolute best out there for media serving.
But it only shows the Categories "Movies" and "Series". So I can't watch my self edited Videos on Findroid.
The best Feature of Findroid is the easy download option, and I would like that to have for my other Media types too.
I tried Jellyfins App for Streaming these Videos, but the Client is slow and buggy when you used to Findroid.

**Describe the solution you'd like**
I would like to have the Option to toggle, which Library types are shown in the Options Menu, as I think not everybody wants that feature, but those who want, can toggle it on. In particular, the Category "Home videos" (that's how it's called in German)

**Describe alternatives you've considered**
I tried the Jellyfin Client, but for me, it's frustrating, because I'm used to the smooth Experience of Findroid.
Additionally, there is no in App download Option, which Findroid has.

**Additional context**
-

**Comment by vanontom** [comment]
Agreed. My "Home videos and photos" (Videos) library type does not appear in Findroid. Directory contains various folders and videos (video and audio samples, tests, everything I don't want in Movies or Shows). Would love for it to be usable in Findroid.

**Comment by nanqiangyipo** [comment]
For the video formats currently supported, no modifications are needed; for other types, the ability to cache and play is sufficient.
